### ISSUE TITLE
missing parameter OSM2PGSQL_EXTRA_ARGS

### ISSUE THREAD

**Issue by stevo01** [Overv/openstreetmap-tile-server#64]
I added the parameter OSM2PGSQL_EXTRA_ARGS for call of osm2psql in script openstreetmap-tiles-update-expire. I used a patch to solve the problem and tested the feature. 

Description of the test sequence: 
https://github.com/OpenSeaMap/osm-tileserver/blob/master/Doc/ImportGermany.md

patch:
```
diff --git a/openstreetmap-tiles-update-expire b/openstreetmap-tiles-update-expire
index dea6e9e..6958f64 100644
--- a/openstreetmap-tiles-update-expire
+++ b/openstreetmap-tiles-update-expire
@@ -22,7 +22,7 @@ OSM2PGSQL_BIN=osm2pgsql
 TRIM_BIN=/home/$ACCOUNT/src/regional/trim_osc.py
 
 DBNAME=gis
-OSM2PGSQL_OPTIONS="-d $DBNAME -G --hstore --tag-transform-script /home/renderer/src/openstreetmap-carto/openstreetmap-carto.lua -C 2048 --number-processes ${THREADS:-4} -S /home/renderer/src/openstreetmap-carto/openstreetmap-carto.style"
+OSM2PGSQL_OPTIONS="-d $DBNAME -G --hstore  ${OSM2PGSQL_EXTRA_ARGS} --tag-transform-script /home/renderer/src/openstreetmap-carto/openstreetmap-carto.lua -C 2048 --number-processes ${THREADS:-4} -S /home/renderer/src/openstreetmap-carto/openstreetmap-carto.style"
 
 #------------------------------------------------------------------------------
 # When using trim_osc.py we can define either a bounding box (such as this
```

**Comment by Overv** [comment]
Could you submit the patch as a pull request?

**Comment by stevo01** [comment]
solved with a2eceb6bb50590186673537cc05fe6099708d2f2
